src/lib/scheduler/abstract_task.cpp

```diff
@@ -115,15 +114,13 @@ void AbstractTask::schedule(NodeID preferred_node_id) {
 }
```

Comment:
Can only mark here: change `_done_condition_variable` to `_task_done` variable in L 105/106.